src/Presentation/RestApi/Resources/config/routing.yaml

```diff
@@ -0,0 +1,9 @@
+api_v1_index_index:
```

Comment:
incorrect route name. Must be `index`. Also, `api_v1` should be moved above as a route prefix. More info -> https://symfony.com/doc/current/routing.html#route-groups-and-prefixes

the same for the route below.